model/LoggerService.php

```diff
@@ -65,7 +67,7 @@ public static function logEvent(Event $event)
             $event->getName(),
             $context->getRequest()->getRequestURI(),
             $currentUser->getIdentifier(),
-            join(',', $currentUser->getRoles()),
+            join(',', $currentUser->getPropertyValues(PROPERTY_USER_ROLES)),
             (new DateTime())->format(DateTime::ISO8601),
```

Comment:
May better use tao_helpers_Date to keep date formats consistent over application ?